app/views/searches/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form_tag(root_path, :method => "get") do %>
+  <div class="row collapse">
+    <div class="small-10 columns">
+      <%= text_field_tag :search_item, params[:search_item], placeholder: "Search Recipes" %>
+    </div>
+  <div class="button small small-2 columns"> <%= submit_tag "Search", :name => nil %></div>
+  </div>
+<% end %>
+
+
+<% if @page_results %>
+  <section class="row small-up-1 medium-up-2 large-up-5">
+    <% @page_results.each do |result| %>
```

Comment:
While this certainly works, I don't know that I'm a big fan of reusing the search results view for the landing page. I would argue these are separate functions and ought to have separate routes and controller actions.